spec/models/cart_spec.rb

```diff
@@ -0,0 +1,7 @@
+require 'rails_helper'
+
+RSpec.describe Cart, type: :model do
+  it "can hold items" do
+      assert Cart.new({"2" => 3})
```

Comment:
Redundant curly braces around a hash parameter.<br>Use 2 (not 4) spaces for indentation.<br>Space inside { missing.<br>Space inside } missing.